orchestra/actions/util/__init__.py

```diff
@@ -105,21 +105,23 @@ def get_subprocess_output(
     environment=None,
     decode_as="utf-8",
     cwd=None,
+    check_returncode=True,
 ):
     """
     Helper to run a subprocess and get its output.
-    If the subprocess returns a nonzero exit code an error is logged and an OrchestraException is raised.
     :param argv: the argv passed to subprocess.run
     :param environment: environment variables
     :param decode_as: decode the output using this encoding
     :param cwd: if not None, the command is executed in the specified path
+    :param check_returncode: by default if the subprocess returns a nonzero exit code an error is logged and an
+            InternalSubprocessException is raised. Setting check_returncode=False overrides this behaviour
```

Comment:
We spent quite some time in trying to cleanup the API to spawn external process, in particular to get rid of boolean arguments.

Let's be consistent and make a `try_` method.

Reply:
Now `get_script_output` (and analogues) are always used in all places where the script must either return an output and return code 0 or return an error code (in that case, an exception is raised and caught where appropriate).
The `try_*` version instead always returns the child's output, and does not check nor return the exit code.

Reply:
Uhm, so there's no way to understand if it failed or not? Shouldn't we return a pair with the exit code?

Reply:
Returning a pair is a good idea, I'll do that